Editor/MenuInstallHook.cs

```diff
@@ -45,6 +46,27 @@ public void OnPreprocessAvatar(GameObject avatarRoot, BuildContext context)
                 avatar.expressionsMenu = menu;
                 context.ClonedMenus[menu] = menu;
             }
+            else
+            {
+                IEnumerable<VRCExpressionsMenu> GetMenus(VRCExpressionsMenu menu)
+                {
+                    if (!menu) yield break;
+                    foreach (var m in menu.controls
+                        .Where(control => control.type == VRCExpressionsMenu.Control.ControlType.SubMenu)
+                        .SelectMany(control => GetMenus(control.subMenu)))
+                        yield return m;
+                    yield return menu;
+                }
```

Comment:
This will result in infinite recursion if there is a menu loop (this is not a theoretical concern; it's been the cause for bug reports in the past)